frontend/src/components/lib/components/Dialog/Dialog.tsx

```diff
@@ -43,7 +43,7 @@ export const Dialog = (props: DialogProps) => {
           <Text text={title} type="h1" />
           <CloseIcon onClick={closeDialog} color={buttonColor} sx={{ cursor: 'pointer' }} />
         </Container>
-        <Container>{children}</Container>
+        <Container paddingX="16px">{children}</Container>
```

Comment:
why

Reply:
This is so that the title of the dialog and its content are on the same line.

![Screenshot 2023-06-11 at 13 47 32](https://github.com/deepchecks/monitoring/assets/42544127/98ef178d-4d58-46a9-bbc2-995b9f03cf9c)
![Screenshot 2023-06-11 at 13 48 13](https://github.com/deepchecks/monitoring/assets/42544127/154cbd04-6ee4-4d28-b30a-20ee76f63155)

Reply:
We also can change the padding of the dialog's header container.

![Screenshot_1](https://github.com/deepchecks/monitoring/assets/42544127/4e934641-013d-41af-ac48-bcef48e6c797)